nginx.default.conf

```diff
@@ -0,0 +1,35 @@
+upstream app_cluster {
```

Comment:
I'd suggest placing this under build/docker directory. It has the relevant assets for docker related stuff.

Reply:
Thought about that, wasn't sure if the build directory was supposed to only hold non dev release stuff. WIll move there